test/README.md

```diff
@@ -0,0 +1,44 @@
+# Linear Vesting Tests Documentation
+
+This offers a comprehensive guide to the Linear Vesting tests within the project. The property based tests (in `./LinearVestingSpec.hs`) are meticulously designed to assess the accuracy of Linear Vesting implementation, particularly focusing on the validation of actions involved during the unlocking phase of vesting.
+
+## Overview
+
+The property based tests for the Linear Vesting are organized into a suite titled "Linear Vesting Tests", This suite is dedicated to verifying the validation process for the accurate implementation of Linear Vesting Contracts.
+
+## Test Suite Details
+
+### Tests Included
+
+-**Partial Unlock**: This property based test single-handedly confirms the proper initialization of Linear Vesting Contract and the successful unlocking of vested assets from the Contract at correct intervals. It does so by programmatically generating **100** test scenarios with entirely different Vesting datum values, unlock timings and their corresponding vested asset quantities. This confirms that the Contract allows accurate vesting of assets and their claim under varied conditions.
```

Comment:
```suggestion
-**Partial Unlock**: This property based test single-handedly confirms the proper initialization of Linear Vesting Contract and the successful unlocking of vested assets from the Contract at correct intervals. It does so by programmatically generating **100** test scenarios with different Vesting datum values per test run, unlock timings and their corresponding vested asset quantities. Since the whole test suit is ran automatically with each commit, this means that thousands (if not millions) of test cases are tried during the whole lifetime of the project. This confirms that the Contract allows accurate vesting of assets and their claim under varied conditions.
```